vendor/jasmine-only/jasmine-only.js

```diff
@@ -0,0 +1,98 @@
+/* jasmine-only - 0.1.1
+ * Exclusivity spec helpers for jasmine: `describe.only` and `it.only`
+ * https://github.com/davemo/jasmine-only
+ */
+(function() {
```

Comment:
I think the idea would be to add a PR with the ES2015 version, that runs in Node4, and then also cross compile to ES5 to avoid breaking backwards compatibility :)